src/hooks/useAuth.js

```diff
@@ -33,6 +34,13 @@ const useAuth = () => {
     };
   }, []);
 
+  React.useEffect(() => {
```

Comment:
I don't think this is working. What do you think of this https://github.com/jrmi/airboardgame/pull/242

Reply:
Let's go with #242 !